README.md

```diff
@@ -1,26 +1,31 @@
 # KlaviyoSwift
 
 [![CI Status](https://travis-ci.org/klaviyo/klaviyo-swift-sdk.svg?branch=master)](https://travis-ci.org/klaviyo/klaviyo-swift-sdk)
+[![Swift](https://img.shields.io/badge/Swift-5.3_5.4_5.5_5.6-orange?style=flat-square)](https://img.shields.io/badge/Swift-5.3_5.4_5.5_5.6-Orange?style=flat-square)
+[![Swift Package Manager](https://img.shields.io/badge/Swift_Package_Manager-compatible-orange?style=flat-square)](https://img.shields.io/badge/Swift_Package_Manager-compatible-orange?style=flat-square)
 [![Version](https://img.shields.io/cocoapods/v/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
 [![License](https://img.shields.io/cocoapods/l/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
 [![Platform](https://img.shields.io/cocoapods/p/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
 
-## Overview
 
-KlaviyoSwift is an SDK, written in Swift, for users to incorporate Klaviyo's event tracking functionality into iOS applications. We also provide an SDK written in [Objective-C](https://github.com/klaviyo/klaviyo-objc-sdk). The two SDKs are identical in their tracking functionality. **However, the KlaviyoSwift SDK is the only one that supports push notifications**. We strongly encourage the use of the KlaviyoSwift SDK.
+## Overview
 
-## Requirements
-- iOS 9.0+ 
-- Xcode 10.2+
-- Swift 5.0+
+KlaviyoSwift is an SDK, written in Swift that can be integrated into your iOS App. This will allow you to message your users via receive push notificaitons from Klaviyo. In addition you will be able to take advanatage of Klavaiyo's identificaiton and event tracking funtionality. Once integrated your marketing will be able to better understand your app users needs and send them timely messages via APNS.
 
 ## Installation Options
 
-1. CocoaPods (recommended)
-2. Download a blank, pre-configured project, and get started from scratch. 
-3. Download the zip file, and drag and drop the KlaviyoSwift file into your project. 
+1. SPM 
+1. CocoaPods
+
+## SPM
+KlaviyoSwift is available via [Swift Package Manager (SPM)](https://swift.org/package-manager/). Follow the steps below to get it setup.
+
+### Import the SDK
+Open your project and navigate to your project’s settings. Select the Swift Packages tab and click on the  add button below the packages list. Enter the URL of our Swift SDK repository (https://github.com/klaviyo/klaviyo-swift-sdk) in the text field and click Next. On the next screen, select the SDK version (1.6.0 as of this writing) and click Next.
```

Comment:
There are 2 spaces between "the" and "add" in "click on the add button below the packages list."